client/src/components/ImageUpload/ImageUploadButton.js

```diff
@@ -0,0 +1,169 @@
+    box-shadow: 2px 5px 20px rgba(0, 0, 0, 0.10);
+    border-radius: 25px;
+    padding: 1rem;
+    border: none;
+    cursor: pointer;
+    color: white;
+    font-family: 'Manrope';
+
+
+    :hover{
+        box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.25);
+    }
+    
+    margin-left:1rem;
+    margin-top: 1rem;
+`;
+
+const Input = styled.input`
+	width: 0.1px;
+	height: 0.1px;
+	opacity: 0;
+	overflow: hidden;
+	position: absolute;
+	z-index: -1;
+`;
+
+const Label = styled.label`
+    background-color: ${(props) => props.theme.colors.red};
+    box-shadow: 2px 5px 20px rgba(0, 0, 0, 0.10);
+    border-radius: 25px;
+    padding: 1rem;
+    border: none;
+    cursor: pointer;
+    display: inline-block;
+    text-align: center;
+    align-items: flex-start;
+    text-indent: 0px;
+
+    :hover{
+        box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.25);
+    }
+    
+    margin-left:1rem;
+    margin-top: 1rem;
+`;
+
+const RowHeader = styled(TagHeader)`
+  margin: 0;
+  width: 15rem;
+  @media screen and (max-width: 801px) {
+    width: auto;
+    max-width: 85%;
+  }
+`;
+
+const TextWrapper = styled.div`
+    font-style: normal;
+    font-weight: 500;
+    color: ${(props) => props.theme.colors.white};
+    text-align: center;
+    line-height: 21px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+`;
+
+const Word = styled.div`
+    font-size:18px;
+`;
+
+const ImageUploadButton = ({clubProfileId, clubProfile, setClubProfile}) => {
+    const [fileURL, setFileURL] = useState(clubProfile.imageUrl);
+
+    useEffect(() => {
+        setFileURL(clubProfile.imageUrl);
+    }, [clubProfile]);
+
+    //Brainstorm error handler for this
+
+    const fileUploadHandler = (event) => {
+        setFileURL(URL.createObjectURL(event.target.files[0]));
+
+        const fd = new FormData();
+        fd.append('image', event.target.files[0], event.target.files[0].name);
+
+        fetch(`/api/image-upload//imgUpload/${clubProfileId}`, {
+            method: 'POST',
+            body: fd
+        })
+        .then(res => res.json())
+        .then(data => {
+            console.log(data);
+            setClubProfile({...clubProfile, imageUrl: data.imageUrl});
+        })
+        .catch(err => {
+            console.error(err);
+        });
+    };
+
+    return (
+        <Wrap>
+            <RowHeader>
+                <RedAsterisk>*</RedAsterisk>
+                Upload a Logo for Your Club:
+            </RowHeader>
+            <ImageBox>
+                <Preview src = {fileURL}></Preview>
+            </ImageBox>
+            <ButtonWrapper>
+                <Input 
+                    type="file" 
+                    name="file" 
+                    onChange={fileUploadHandler}
+                    id="file" 
+                    class="inputfile" 
+                />
+                <Label for="file">
+                    <TextWrapper><Word>Select Image</Word></TextWrapper>
```

Comment:
modified this to only require one button-- i kept forgetting to hit upload so i figured a user probably would too haha